Stack/Opc.Ua.Core/Security/Certificates/CertificateValidator.cs

```diff
@@ -709,12 +709,9 @@ private X509SubjectKeyIdentifierExtension FindSubjectKeyIdentifierExtension(X509
                 throw new ServiceResultException(StatusCodes.BadSecurityChecksFailed, "SHA1 signed certificates are not trusted");
             }
 
-            using (RSA rsa = certificate.GetRSAPublicKey())
+            if (certificate.GetRSAPublicKey().KeySize < m_minimumCertificateKeySize)
```

Comment:
No need to dispose here?

Reply:
Having the public key not immediately disposed seems ok here...

---
In reply to: [163232893](https://github.com/OPCFoundation/UA-.NETStandard/pull/325#discussion_r163232893) [](ancestors = 163232893)